buster/formatter/base.py

```diff
@@ -0,0 +1,63 @@
+from dataclasses import dataclass
+from typing import Iterable, NamedTuple
+
+# Should be from the `documents` module.
+class Source(NamedTuple):
+    name: str
+    url: str
+    question_similarity: float
+    # TODO Add answer similarity.
+    # answer_similarity: float
+
+
+# Should be from the `nlp` module.
+@dataclass(slots=True)
+class Response:
+    text: str
+    error: bool = False
+    error_msg: str | None = None
+
+    def __bool__(self) -> bool:
+        return not self.error
```

Comment:
```suggestion```
This should delete the function.